kani-driver/src/cbmc_output_parser.rs

```diff
@@ -0,0 +1,989 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Module for parsing CBMC's JSON output. In general, this output follows
+//! the structure:
+//! ```
+//! [
+//!     Program,
+//!     Message,
+//!     ...,
+//!     Message,
+//!     Result,
+//!     Message,
+//!     ProverStatus
+//! ]
+//! ```
+//! The parser included in this file reads from buffered input line by line, and
+//! determines if an item can be processed after reading certain lines.
+//!
+//! The rest of code in this file is related to result postprocessing.
+
+use crate::{args::OutputFormat, call_cbmc::VerificationStatus};
+use anyhow::Result;
+use console::style;
+use pathdiff::diff_paths;
+use regex::Regex;
+use serde::Deserialize;
+use std::{
+    collections::HashMap,
+    env,
+    io::{BufRead, BufReader},
+    path::PathBuf,
+    process::{Child, ChildStdout},
+};
+use structopt::lazy_static::lazy_static;
+
+lazy_static! {
```

Comment:
I do not want this to block this PR, but IIRC `lazy_static` isn't really maintained anymore and I forget what crate is supposed to replace it. `once` or `oncecell` or something like that.